packages/drivechain_client/lib/widgets/address_menu.dart

```diff
@@ -0,0 +1,224 @@
+import 'package:drivechain_client/pages/send_page.dart';
+import 'package:flutter/material.dart';
+import 'package:get_it/get_it.dart';
+import 'package:stacked/stacked.dart';
+import 'package:drivechain_client/address_book.dart';
+import 'package:sail_ui/sail_ui.dart';
+
+class AddressMenu extends StatelessWidget {
+  const AddressMenu({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ViewModelBuilder<AddressMenuViewModel>.reactive(
+      viewModelBuilder: () => AddressMenuViewModel(),
+      builder: (context, model, child) {
+        return Column(
+          children: [
+            SailText.primary12(
+              'These are your Bitcoing addresses for sending payments. Always check the amount and the receiving address before sending coins.',
```

Comment:
typo in bitcoing

Reply:
never heard of bitcoing, what's a bitcoing?

Reply:
1 bitcoing is the equivalent of 100,000,000 satoshing

Reply:
have never heard of this before

Reply:
maybe for denominating amounts, but i'm pretty sure it's not used to refer to addresses